mkdocs/themes/readthedocs/base.html

```diff
@@ -84,36 +85,36 @@
     <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
     <div class="wy-side-scroll">
       <div class="wy-side-nav-search" {% if config.theme.style_nav_header_background %} style="background: {{config.theme.style_nav_header_background}}" {% endif %}>
-    {%- block site_name %}
-        <a href="{{ nav.homepage.url|url }}" class="icon icon-home"> {{ config.site_name }}</a>
-	  {%- endblock %}
-	  {%- block search_button %}
+      {%- block site_name %}
+        <a href="{{ nav.homepage.url|url }}" class="icon icon-home" alt="{% trans %}Docs{% endtrans %}"> {{ config.site_name }}</a>
```

Comment:
I believe I left it as `Docs` because that's what MkDocs had and I didn't want to change the `msgid` in the `messages.pot` file causing all the previous translation work to be null for that key.